### PR TITLE
[Aptos Tournament] Processor skeleton

### PR DESCRIPTION
**Pull request by just-in-chang**
Skeleton for processor to index Aptos Tournament

### CODE REVIEW

File: rust/processor/migrations/2023-11-10-004930_create_aptos_tournament_tables/up.sql [hunk 1]
```diff
@@ -0,0 +1,45 @@
+-- Create tournaments table
+CREATE SCHEMA IF NOT EXISTS aptos_tournament;
```
Comment (bowenyang007) [inline rust/processor/migrations/2023-11-10-004930_create_aptos_tournament_tables/up.sql:2]: keep it in public is fine.

File: rust/processor/src/models/aptos_tournament_models/aptos_tournament_utils.rs [hunk 2]
```diff
@@ -0,0 +1,457 @@
...
+
+/// Tracks all tournament related data in a hashmap for quick access (keyed on address of the object core)
+// Do I need this lol.. i'm just copying token v2
+pub type AptosTournamentAggregatedDataMapping = HashMap<String, AptosTournamentAggregatedData>;
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct AptosTournamentAggregatedData {
+    pub aptos_tournament: Option<AptosTournament>,
+    pub current_round: Option<CurrentRound>,
+    pub refs: Option<Refs>,
```
Comment (bowenyang007) [inline rust/processor/src/models/aptos_tournament_models/aptos_tournament_utils.rs:18]: u dont need this

File: rust/processor/src/models/aptos_tournament_models/players.rs [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+impl Player {
+    pub async fn upsert(
+        &self,
+        conn: &mut PgPoolConnection<'_>,
+    ) -> Result<usize, diesel::result::Error> {
+        conn.build_transaction()
+            .read_write()
+            .run::<_, Error, _>(|pg_conn| Box::pin(self.upsert_impl(pg_conn)))
+            .await
+    }
```
Comment (bowenyang007) [inline rust/processor/src/models/aptos_tournament_models/players.rs:36]: why copy this function? just keep it in the processor file

**Review by bowenyang007 — CHANGES_REQUESTED** [review]
close?

**Comment by just-in-chang** [comment]
dont delete branch in case needed for future reference

**Pull request closed without merging**